### PR TITLE
(packaging) Update version to 2.11.0 for release

### PR DESCRIPTION
**Pull request by McdonaldSeanp**

### CODE REVIEW

**Comment by Magisus** [comment]
This agrees with the version stated in https://confluence.puppetlabs.com/display/PM/puppet-agent+Versions+and+Dependencies for the PA 5.0.0 release 👍

File: website/releasenotes.md [hunk 1]
```diff
@@ -10,6 +10,21 @@ will be highlighted here.
 
 <a name="2_10_5">&nbsp;</a>
```
Comment (MikaelSmith) [inline website/releasenotes.md]: I think this needs to be updated as well.

**Comment by McdonaldSeanp** [comment]
@MikaelSmith updated

File: website/releasenotes.md [hunk 2]
```diff
@@ -8,6 +8,21 @@ This is a list of release notes for various releases, you should review these
 before upgrading as any potential problems and backward incompatible changes
 will be highlighted here.
 
+<a name="2_11_0">&nbsp;</a>
+
+## 2.11.0 - 2017/06/21
+
+### Changes since 2.10.4
```
Comment (MikaelSmith) [inline website/releasenotes.md]: Should say since 2.10.5, right?

**Pull request merged**